homeassistant/components/mystrom/__init__.py

```diff
@@ -51,12 +71,6 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         _LOGGER.error("Unsupported myStrom device type: %s", device_type)
         return False
 
-    try:
-        await device.get_state()
-    except MyStromConnectionError as err:
-        _LOGGER.error("No route to myStrom plug: %s", info["ip"])
-        raise ConfigEntryNotReady() from err
```

Comment:
I don't see this happening anymore ? This is important logic to make sure we retry if we cannot connect.

Reply:
Oh I see it now, nevermind.